src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -55,6 +58,9 @@ public class DistributedApplicationBuilder : IDistributedApplicationBuilder
     /// <inheritdoc />
     public string AppHostDirectory { get; }
 
+    /// <inheritdoc />
+    public Assembly? AppHostAssembly => _options.Assembly;
```

Comment:
Do we really need to expose this as public API?

Reply:
We can revert but it's kinda useful to have. I think originally it was required when the user secrets stuff was public and each resource extension method was doing the logic to add the user secrets parameter default, but it could all be internal now.

Reply:
Logged #4612